### PR TITLE
Abzu 48428 Static pages for Home, Cumulative, Annual, Leisure, VARs & About (x2)

### PR DESCRIPTION
**Pull request by barrie-cooper**
Quick & dirty static pages for the following which will suffice until the Mastek PBIs are completed, after the initial public release:
- Home
- Cumulative NMs
- Annual NMs
- Leisure Folios
- Value Added Resellers
- About NMs
- About RNWs

Also incorporates fixing the incorrect terms used in the prototype ([PBI 48321](https://dev.azure.com/ukhydro/Maritime%20Safety%20Information/_workitems/edit/48321)):
- `Notice to Mariners` -> `Notices to Mariners`
- `Radio Navigation Warnings` -> `Radio Navigational Warnings`

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                    <td>E W Liner Charts & Publications</td>
+					<td><a href="https://www.ewliner.com" target="_blank">www.ewliner.com</a></td>
+                </tr>
+                <tr>
+                    <td>Easy Marine</td>
+					<td><a href="https://www.easymarine.it" target="_blank">www.easymarine.it</a></td>
+                </tr>
+                <tr>
+                    <td>Guangzhou Haining Maritime Service</td>
+					<td>---</td>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml:85]: Existing MSI application displaying N/A for website for this VAR. Not sure if we keep it same?
```suggestion
					<td>N/A</td>
```

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                    <td>Qija Voyage Ltd</td>
+					<td><a href="http://www.qjvoy.net/en/" target="_blank">www.qjvoy.net</a></td>
+                </tr>
+                <tr>
+                    <td>SEEKARTE Kapitän A. Dammeyer GmbH & Co. KG</td>
+					<td><a href="https://www.seekarte.de" target="_blank">www.seekarte.de</a></td>
+                </tr>
+                <tr>
+                    <td>Shanghai Ocean Nautical Supply Company</td>
+                    <td>---</td>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml:153]: Same as above comment. Existing MSI application displaying N/A for website for this VAR. Not sure if we keep it same?
```suggestion
                    <td>N/A</td>
```

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+                </tr>
+                <tr>
+                    <td>Shenzhen Goldsailocean Navigation Article Co Ltd</td>
+                    <td>---</td>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml:157]:
```suggestion
                    <td>N/A</td>
```

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    ViewData["Header"] = "About";
+}
+<h4>
+    ADMIRALTY Notices to Mariners (NMs)
+</h4>
+<p>
+    ADMIRALTY charts and publications should be maintained so that they are fully up-to-date for the latest safety-critical navigational information. The ADMIRALTY Notices to Mariners service provides all the data you need to maintain our products.
+</p>
+<p>
+    ADMIRALTY NMs contain all the corrections, alterations and amendments for the UKHO's worldwide series of ADMIRALTY charts and publications. They are published weekly in several formats but best known as the weekly booklets, which are dispatched directly from the UKHO.
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml:14]: Not sure if we correct it or keep it same as existing?

```suggestion
    ADMIRALTY NMs contain all the corrections, alterations and amendments for the UKHO's worldwide series of ADMIRALTY charts and publications. They are published weekly in several formats but best known as the weekly booklets, which are despatched directly from the UKHO.
```

**Review by usha13529 — COMMENTED** [review]
Few observations to address please, otherwise all changes looks good. Happy to approve.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+        Cumulative List of ADMIRALTY Notices to Mariners (NP234A and NP234B) - £6.00 per copy
+    </li>
+    <li>
+        Annual Summary of ADMIRALTY Notices to Mariners (NP247) (paper version - parts 1 and 2) - £8.90 per copy
+    </li>
+    <li>
+        Notices to Mariners Web Services - Free
+    </li>
+    <li>
+        Notices to Mariners Web Services - Free
```
[the remark this answers is not in the document]
Comment (barrie-cooper) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml:104]: Good spot, thanks

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml (hunk 4 above)
Comment (barrie-cooper) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml:14]: I just checked...`dispatch` is the American spelling and `despatch` is the British one so I've just leaned something :)

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml (hunk 1 above)
Comment (barrie-cooper) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml:85]: Okay, I preferred `---` but not too worried either way.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml (hunk 3 above)
Comment (barrie-cooper) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Resellers.cshtml:157]: Done

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Leisure.cshtml [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+    Here you will find all relevant ADMIRALTY Notices to Mariners (NMs) released since the publication date of the chart within the leisure range's New Edition. This service is updated weekly and aims to provide you with the best information available to help you navigate safely and effectively. You can be assured that with ADMIRALTY charts you are using the best; they are recognised official standard and trusted worldwide. We pride ourselves on the quality and integrity of our data - no one else promulgates information as well or as quickly as we do.
+</p>
+<p>
+    For further information, guidance notes including the printing of blocks, please click <a href="https://www.admiralty.co.uk/AdmiraltyDownloadMedia/ADMIRALTY%20Leisure%20Charts/Guidance/Keeping%20ADMIRALTY%20Leisure%20Charts%20up-to-date.pdf" target="_blank">here</a>.
+</p>
+
+<div class="alert alert-primary p-4" role="alert">
+    <i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>
+    <div class="alert-head">
+        Important information
```
[the remark this answers is not in the document]
Comment (barrie-cooper) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Leisure.cshtml:20]: Again, good spot, thanks

**Review by usha13529 — APPROVED** [review]
Thank you for the changes @barrie-cooper. Looks good to me. Thanks.

**Review by crdandoukho — APPROVED** [review]

**Pull request merged**